Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM golang:1.16-alpine
+
+RUN apk update && apk add curl && apk add --no-cache bash
+WORKDIR /app
+
+COPY go.sum ./
+COPY .env ./
+COPY go.mod ./
+RUN go mod download
+
+COPY *.go ./
+# TODO just pull down mirth-cli-launcher instead of entire mirth server, etc.
+# running into AWS access denied errors, even locally when logged in through aws-value or sso
+RUN #curl -SL 'https://mirth-client-library.s3.amazonaws.com/mirth-cli-launcher.jar'
```

Comment:
running into AWS access denied errors, even locally when logged in through aws-value or sso